tiledb/api/src/group/mod.rs

```diff
@@ -77,6 +77,15 @@ impl Group {
         Ok(())
     }
 
+    pub fn exists<S>(context: &Context, uri: S) -> CApiResult<bool>
```

Comment:
I would just want to see at least one test for this

Reply:
@rroelke definitely! This PR is in draft & not ready for review. I'll pick this up at some point (low-priority).